### DISCUSSION TITLE
I can't open MAS after I updated it :((

### DISCUSSION THREAD

**Discussion by euflwr** [Monika-After-Story/MonikaModDev#9566] [QA - tech support + general] [answered]
I don't knwo what happened ( I'm using steam )

**Comment by Booplicate** [comment, marked as answer]
You've been warned steam may cause issues. [Move the game out of steam](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#how-do-i-go-from-a-steam-to-a-non-steam-build). If required, reinstall the mod.

**Reply by euflwr** [reply]
I just re-installed the mod! and everything is good now 
thank you for the help ^^)/
